drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -254,6 +271,17 @@ end
 local function device_init(driver, device)
   device:subscribe()
   device:set_component_to_endpoint_fn(component_to_endpoint)
+
+  if not device:get_field(setpoint_limit_device_field.MIN_SETPOINT_DEADBAND_CHECKED) then
+    local auto_eps = device:get_endpoints(clusters.Thermostat.ID, {feature_bitmap = clusters.Thermostat.types.ThermostatFeature.AUTOMODE})
+    --Query min setpoint deadband if needed
+    if #auto_eps ~= 0 and device:get_field(setpoint_limit_device_field.MIN_DEADBAND) == nil then
+      local setpoint_limit_read = im.InteractionRequest(im.InteractionRequest.RequestType.READ, {})
```

Comment:
This should just be `deadband` read now should it?

Reply:
Changed as suggested!